ajax/libs/reframe.js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "reframe.js",
+  "filename": "reframe.min.js",
+  "version": "2.1.14",
+  "description": "Reframe.js: responsive iframes for embedded content",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dollarshaveclub/reframe.js.git"
+  },
+  "keywords": [
+    "js",
+    "iframes",
+    "fluid",
+    "responsive",
+    "video",
+    "fitvids",
+    "youtube",
+    "embeds",
+    "padding-top"
+  ],
+  "author": "Jeff Wainwright <jjwainwright2@gmail.com> (http://jeffry.in)",
+  "homepage": "https://dollarshaveclub.github.io/reframe.js/",
+  "npmName": "reframe.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|css)"
```

Comment:
Let's use `**/!(*.scss)` here. In kind of `dist` folders, cdnjs would like to be more flexible to fetch files. So usually excluding files would be better choice.

Please leave a comment in PR to let reviewers know why you use some special rules to speed up review. Thanks.